File: 2942_Find_Words_Containing_Character.cpp
class Solution {
public:
    vector<int> findWordsContaining(vector<string>& words, char x) {
        vector<int> h;
        for (int i = 0; i < words.size(); ++i) {
            bool d = false;
            for (char c : words[i]) {
                if (c == x) {
                    d = true;
                    break;
                }
            }
            if (d) {
                h.push_back(i);
            }
        }
        return h;
    }
};

File: 2942_Find_Words_Containing_Character.java
class Solution {
    public List<Integer> findWordsContaining(String[] words, char x) {
        List<Integer> v = new java.util.ArrayList<>();
        for (int i = 0; i < words.length; ++i) {
            if (words[i].indexOf(x) != -1) {
                v.add(i);
            }
        }
        return v;
    }
}
